scripts/teahouse/send_th_invites.py

```diff
@@ -40,20 +40,27 @@ def getEligibleInvitees(elig_check, potential_invitees):
     eligible_invitees = [x for x in potential_invitees if elig_check.determineInviteeEligibility(x)]
     return eligible_invitees
 
-def runSample(c, inviter, message, params):
-    prof = hb_profiles.Profiles(params['output namespace'] + c[0], user_name = c[0], user_id = c[1], page_id = c[2],  settings = params)
-    prof.inviter = inviter
-    prof.message = message
+def runSample(invitee, inviter, params):
+    prof = hb_profiles.Profiles(
+        params['output namespace'] + invitee[0],
+        user_name = invitee[0],
+        user_id = invitee[1],
+        page_id = invitee[2],
+        settings = params
```

Comment:
was `settings`/`profile_settings` ever used? If not this would be a good time to drop it.